bot.js

```diff
@@ -152,6 +149,26 @@ client.on('message', message => {
 		});
 	}
 
+	if(message.content.substring(0,6) === `${prefix}class`){
+		const args = message.content
+		const req = args.split(" ").length === 3 ? `${args.split(" ")[1]}${args.split(" ")[2]}`:args.split(" ")[1]
```

Comment:
`const req = args.slice(1).join('')`

Reply:
This does not work

Reply:
whoops, left a part out

Reply:
still need to split it, but just discard the first index, and join everything after